src/store/pageview.ts

```diff
@@ -17,6 +26,29 @@ class ViewStore {
     storage.set(StorageKeys.PAGE_VIEWS, this.pageViews);
   }
 
+  _trim() {
+    this.pageViews.sort((a: PageView, b: PageView): number => b.ts - a.ts);
+    this.pageViews = this.pageViews.filter(
+      (pageView: PageView, i: number) =>
+        pageView.ts > timeStampInSecs() - this.maxAge && i < this.storageSize
+    );
+  }
+
+  setMaxAge(maxAge?: number) {
+    if (!maxAge || maxAge < 0 || maxAge === this.maxAge) return;
+    this.maxAge = maxAge;
+    this._trim();
+    this._save();
```

Comment:
We cannot call this twice for each setting of these values.

Reply:
It is only executed on the first set of each, but yes, it is being called twice at that time.
Would you rather have a single setter for both values?
It is a valid choice, indeed.

Reply:
Yeah I would set it from one method.

Reason: I do not see a use case for them being set independetly, so this was we reduce the size of the API and therefore complexity and potential bugs :)

Reply:
I agree.
Will change.